### PR TITLE
6742 - Changed organization terms

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#6742
*Description of changes:*

- Changed organization terms; 
- Fixed tests.

** PLEASE DO NOT MERGE YET **
Legal-api changes need to be in sync.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/incorporatorTypes.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export enum IncorporatorTypes {
     PERSON = 'person',
-    CORPORATION = 'org'
+    CORPORATION = 'organization'
```
Comment (severinbeauvais) [inline src/enums/incorporatorTypes.ts:3]: Would you mind renaming IncorporatorTypes.CORPORATION to IncorporatorTypes.ORGANIZATION?

**Review by severinbeauvais — APPROVED** [review]
LGTM. I hope all UIs are just as easy :)

File: src/enums/incorporatorTypes.ts (hunk 1 above)
Comment (andrepestana-aot) [reply src/enums/incorporatorTypes.ts:3]: Sure. Will do it.

Comment (severinbeauvais) [reply src/enums/incorporatorTypes.ts]: Thanks!

File: tests/unit/ListPeopleAndRoles.spec.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('List People And Roles component', () => {
         'firstName': 'Cameron',
         'lastName': 'Bowler',
```
Comment (cameron-freshworks) [inline tests/unit/ListPeopleAndRoles.spec.ts:12]: Eternalized in the unit tests!

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

File: tests/unit/ListPeopleAndRoles.spec.ts (hunk 2 above)
Comment (andrepestana-aot) [reply tests/unit/ListPeopleAndRoles.spec.ts:12]: 😆

**Pull request merged**